syz-manager/repro.go

```diff
@@ -205,12 +205,12 @@ func (m *reproManager) adjustPoolSizeLocked() {
 	// Avoid the +-1 jitter by considering the repro queue size as well.
 
 	// We process same-titled crashes sequentially, so only count unique ones.
-	uniqueTitles := make(map[string]bool)
+	uniqueTitles := maps.Clone(m.reproducing)
```

Comment:
Can't comment on line 110, but in onlyOnce mode crashes should be checked against all crashes that were ever reproduced, not just the ones currently in m.queue.
Otherwise a frequent crash will still be reproduced all the time.

Reply:
Thanks.
I'll send it in a separate PR.